$NetBSD: patch-aa,v 1.5 2014/02/04 00:15:14 joerg Exp $

Patch hardcoded with libtcl86.  Previously it was hardcoded to libtcl84.

--- makefile.orig	1998-01-29 11:46:41.000000000 +0000
+++ makefile
@@ -3,12 +3,13 @@ LEX = flex -I -i
 
 YFLAGS		= -dv
 CFLAGS		= -g $(LIBFLAGS) $(IFLAGS) -DSTAND_ALONE
-LIBDIR		= /cs/research/mice/starship/common/solaris/lib
-INCDIR		= /cs/research/mice/starship/common/solaris/include
+LIBDIR		= ${PREFIX}/lib
+INCDIR		= ${PREFIIX}/include
+LIBS            = -ll -ltk86 -ltcl86 -lm -lX11 -L${PREFIX}/lib -Wl,-R${PREFIX}/lib -L${X11BASE}/lib -Wl,-R${X11BASE}/lib $(LDFLAGS)
 
 # The files needed from the tcl/tk library directories...
-LIBRARY_TK	= $(LIBDIR)/tk4.2
-LIBRARY_TCL	= $(LIBDIR)/tcl7.6
+LIBRARY_TK	= $(LIBDIR)/tk8.6
+LIBRARY_TCL	= $(LIBDIR)/tcl8.6
 
 
 # Uncomment these lines for SGI (IRIX 5.3).
@@ -32,10 +33,10 @@ LIBRARY_TCL	= $(LIBDIR)/tcl7.6
 #LIBS		= -ll -ltk4.2 -ltcl7.6 -lm -lX11
 
 # Uncomment these lines for Sun (SunOS 5.5.1).
-CC		= gcc -DFNDELAY=O_NDELAY -DBYTE_ORDER=4321 -ansi
-LIBFLAGS	= -L$(LIBDIR) -L/usr/openwin/lib
-IFLAGS		= -I$(INCDIR) -I/usr/openwin/include
-LIBS		= -ll -ltk4.2 -ltcl7.6 -lm -lX11 -lsocket -lnsl -ldl
+#CC		= gcc -DFNDELAY=O_NDELAY -DBYTE_ORDER=4321 -ansi
+#LIBFLAGS	= -L$(LIBDIR) -L/usr/openwin/lib
+#IFLAGS		= -I$(INCDIR) -I/usr/openwin/include
+#LIBS		= -ll -ltk4.2 -ltcl7.6 -lm -lX11 -lsocket -lnsl -ldl
 
 # Uncomment these lines for NetBSD 1.2.
 #CC              = cc -ansi
